lisa/environment.py

```diff
@@ -146,6 +148,9 @@ def __init__(self, is_predefined: bool, warn_as_error: bool) -> None:
         self.warn_as_error = warn_as_error
         self._default_node: Optional[Node] = None
         self._log = get_logger("env", self.name)
+        # delay construct it until it's used, because the environment name is not set
```

Comment:
I don't understand the context for this comments.

Reply:
Improved the message, hope it's clearer